swagger.yaml

```diff
@@ -173,24 +289,42 @@ definitions:
         type: string
       attributes:
         $ref: "#/definitions/Attributes"
+      links:
+        $ref: "#/definitions/SelfLink"
+      relationships:
+        $ref: "#/definitions/Relationships"
 
-  Attributes:
+  ServicesResultObjects:
     properties:
-      name:
+      links:
+        $ref: "#/definitions/Links"
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/ServicesResourceObject"
+
+  ServicesResourceObject:
```

Comment:
Hmm, the jsonapi.org spec wasn't specific as to what was the preferred way to map back to the locations object. I went with a links.location . It seemed cleaner and simple. We could go with the relationships object, I just don't think it makes sense to have both.